nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/TestCommentsMigrator.java

```diff
@@ -250,6 +253,52 @@ public void testMigrationFromPropertyToSecuredWithCommentsNotCorrectlyFilled() {
         }
     }
 
+    /**
+     * Test the migration in the case where the {@code comment:parentId} is not empty, but it doesn't exist.
+     */
+    @Test
+    @LogCaptureFeature.FilterOn(logLevel = "WARN")
+    @Deploy("org.nuxeo.ecm.platform.comment.tests:OSGI-INF/disable-removing-comment-children-listener.xml")
+    public void testMigrationFromPropertyToSecuredWithRemovedParentComment() {
+        logFeature.hideWarningFromConsoleLog();
```

Comment:
We don't need to hide warning from console because the log capture feature already catch warn logs and this they are not logged.

Reply:
i think the hide warning is required. otherwise (i test with removing the hide) and i get a lot warning logs on my console:

`2020-02-06 09:36:48,404 [main] WARN  [CommentsMigrator] The comment document model with IdRef: 73ffc123-2417-4917-a52a-2c5a0a6fe613 cannot be migrated, because his parent: 0dce638b-c3b4-4bad-8f3a-8d6a496f2c56 cannot be found
`

i think we can keep it  for `10.10` and on master version i will remove it and replace by the annotation configuration

Reply:
Ok for 10.10, didn't pay attention to it as it's in the PR title.

My point is for master, the presence of `LogFeature` on the test without the method nor the annotation is enough to hide log from console. We may have a regression.

Reply:
once the PR validate i will cherry pick and remove the hide console and i will check it

Reply:
but if i remember unfortunately we need the two log to hide and to capture. its why when we rework the log annotation we told ourself we need another JIRA ticket to simply the whole process.
`ConsoleLogLevelThreshold` was added to hide 
`LogCapture` was here before the `ConsoleLogLevelThreshold` and the idea is to create a new jira ticket which will responsible to configure `ConsoleLogLevelThreshold` based on the `LogCapture` without the need to add manually the `ConsoleLogLevelThreshold`

Reply:
Agree about the improvement around these two features. What I'm talking about is that without explicitly hiding, we still hide logs from console on master whereas we shouldn't as the improvement wasn't done.